### PR TITLE
Add 'initial_query_id' field for system.processors_profile_log

### PR DESCRIPTION
**Pull request by helifu**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Facilitate profile data association and aggregation for the same query

### CODE REVIEW

**Comment by CheSema** [comment]
Could you add some simple test here?

**Comment by robot-ch-test-poll3** [comment]
This is an automated comment for commit 489698ae097b148debbb54898a85efd2a399fcf7 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/49777/489698ae097b148debbb54898a85efd2a399fcf7/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🔴 failure</td></tr>
</table>

**Comment by CheSema** [comment]
FYI:
Error: Cannot run, PR description does not match the template: Changelog entry required for category 'Improvement'
https://github.com/ClickHouse/ClickHouse/actions/runs/4947294382/jobs/8871234671?pr=49777

**Comment by helifu** [comment]
> Could you add some simple test here?

done.

File: docs/en/operations/system-tables/processors_profile_log.md [hunk 1]
```diff
@@ -5,16 +5,18 @@ This table contains profiling on processors level (that you can find in [`EXPLAI
 Columns:
 
 - `event_date` ([Date](../../sql-reference/data-types/date.md)) — The date when the event happened.
-- `event_time` ([DateTime64](../../sql-reference/data-types/datetime64.md)) — The date and time when the event happened.
```
Comment (CheSema) [inline docs/en/operations/system-tables/processors_profile_log.md:8]: Thanks for that. event_time is DateTime in the code indeed.

**Review by CheSema — APPROVED** [review]

**Comment by helifu** [comment]
@CheSema
Seems the 'Fast Test' failed, but I don't know what's going on. Could you please give me some tips how to fix this failure?

**Comment by CheSema** [comment]
> @CheSema Seems the 'Fast Test' failed, but I don't know what's going on. Could you please give me some tips how to fix this failure?

Here the report.
https://s3.amazonaws.com/clickhouse-test-reports/49777/ec8f17a71ffa171a817965bc162d265c582fd5ab/fast_test.html

```
2023-05-22 15:23:03 -1
2023-05-22 15:23:03 \ No newline at end of file
2023-05-22 15:23:03 +1
```

When you make a tests, it is better to run it locally and just copy .output file as .reference.

**Pull request merged**